coordination.md

```diff
@@ -2,7 +2,7 @@
 
 Members of Hypha adopt the following practices for mostly asynchoronous coordination using a couple digital platforms.
 
-## Primary coordination
+## Primary Coordination
```

Comment:
We've been inconsistent on title case in the handbook

Reply:
Yes I looked at a couple pages and this page stuck our. This change brings it closer